src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -851,4 +866,54 @@ private String join(Keys[] keys) {
                  .map(n -> n.name())
                  .collect(Collectors.joining("\", \""));
     }
+
+    /**
+     * Checks if a specified text appears on a web site. Even just typed text artifacts 
+     * which are currently inserted into the DOM. 
+     * The expectation is: The user just wants to search for a specified text on a web site 
+     * without an error being reported if the text is not found on the web site. With this
+     * knowledge the user can decide what to do in the test. Using an assertion or just   
+     * assigning the result of this method to a variable for later usage.
+     * 
+     * @param textTobeFound
+     * @return true if searched text is found on a HTML Page and in DOM , false otherwise
+     * @throws FixtureException
+     */
+    @FixtureMethod
+    public boolean isTextOnPage(String textTobeFound) throws FixtureException {
+        // First looking in the page source for fix Web Elements
+        if (checkForConstantWebElements(textTobeFound)) {
+            return true;
+        }
+        // Now check for dynamic Web Elements in DOM 
+        return checkForDynamicWebElements(textTobeFound, getAllWebElementsOnPage());
+    }
+
+    protected List<WebElement> getAllWebElementsOnPage() {
+        return driver.findElements(By.xpath("//*"));
+    }
+
+    protected boolean checkForConstantWebElements(String textTobeFound) {
+        String pageSource = getPageSource();
+        return pageSource.contains(textTobeFound); 
+    }
+
+    protected String getPageSource() {
+        return driver.getPageSource();
+    }
+
+    protected boolean checkForDynamicWebElements(String textTobeFound, List<WebElement> elements) 
+            throws FixtureException {
```

Comment:
Why does this method have to declare `FixtureException` to be thrown? Its not a fixture method itself, and it doesn't seem to actually ever throw the exception.